app/scripts/modules/tencentcloud/src/pipeline/stages/disableCluster/disableClusterStage.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const angular = require('angular');
+import { Registry } from '@spinnaker/core';
+import { DisableCluster } from './DisableCluster';
+
+export const TENCENTCLOUD_PIPELINE_STAGES_DISABLE_CLUSTER = 'spinnaker.tencentcloud.pipeline.stage.disableClusterStage';
+angular.module(TENCENTCLOUD_PIPELINE_STAGES_DISABLE_CLUSTER, []).config(function() {
```

Comment:
You shouldn't need to register any of these stages as Angular modules, as long as each file that registers a stage with the pipeline registry is imported to the tencentcloud entrypoint (`tencentcloud.module.ts`).